variables.tf

```diff
@@ -0,0 +1,114 @@
+##############################################################################
+# Input Variables
+##############################################################################
+
+variable "resource_group_id" {
+  type        = string
+  description = "The resource group name where the Hyper Protect Crypto Service instance will be created."
+}
+
+variable "region" {
+  type        = string
+  description = "The region where you want to deploy your instance."
+}
+
+variable "service_name" {
```

Comment:
Can we name this as just "name" to be in consistent with other modules?  Also service_name indicates as if its a service name in IBM Cloud.

Reply:
Made the required changes.